action.yml

```diff
@@ -27,28 +27,32 @@ runs:
       - name: "Download CI scripts"
         run:  git clone https://github.com/gap-system/pkg-ci-scripts.git
         shell: bash
-      - name: "Install dependencies"
+      - name: "Install dependencies (Linux)"
         run: |
+          if [ ! "${{runner.os}}x" = "Linuxx" ]; then
+            exit 0
+          fi
           if [ "${{inputs.ABI}}x" = "32x" ];
           then
             sudo dpkg --add-architecture i386
             sudo apt-get update
-            packages=(
-              libgmp-dev:i386
-              libreadline-dev:i386
-              zlib1g-dev:i386
-              gcc-multilib
-              g++-multilib
-            )
+            packages=(libgmp-dev:i386 libreadline-dev:i386 zlib1g-dev:i386 gcc-multilib g++-multilib)
```

Comment:
Why? Having each package on its own line seems more readable to me, and if we need to change the list, then the diff is nicer, too